modules/ROOT/pages/6.0-release-notes-events.adoc

```diff
@@ -0,0 +1,100 @@
+= TinyMCE 6.0 Event changes
+:navtitle: TinyMCE 6.0 Event changes
+:description: TinyMCE 6.0 Event changes
+:keywords: releasenotes, events
+
+This section documents the improvements and fixes made to {productname} 6.0 Events.
+
+* xref:events[Events]
+* xref:new-and-improved-events][New and improved Events]
+* xref:fixed-events[Fixed Events]
+
+// tag::events[]
+[[events]]
+== Events
+
+
+[[new-and-improved-events]]
+=== New and improved events
+
+
+[[autocompleter]]
+==== `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd`
+
+As part of moving `autocompleter` from the default `silver` theme to the {productname} core, three new events are now available: `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd`.
+
+Themes can act on the `AutocompleterStart` and `AutocompleterUpdate` events and can call `AutocompleterEnd` to cancel the auto completer.
+
+
+[[before-set-content]]
+==== `BeforeSetContent`
+
+The `BeforeSetContent` event now includes the actual serialized content when passing in an `AstNode` to the `editor.setContent`.
+
+
+[[get-content]]
+==== `GetContent`
+
+The `GetContent` event now always passes a `string` for the `content` property.
+
+
+[[theme-load-error]]
+==== `ThemeLoadError`
+
+A `ThemeLoadError` event is now fired if the theme fails to load.
+
+
+[[fixed-events]]
+=== Fixed events
+
+
+[[before-input]]
+==== `beforeinput`
+
+The `beforeinput` event is expected to fire whenever material is added to or removed from a {productname} instance.
+
+Previously, however, because of overrides in the `DeleteBackspaceKeys.ts` module, the `beforeinput` event did not always fire when expected.
```

Comment:
These are internal files so not something we should be mentioning to customers. Maybe something like this instead?
```suggestion
Previously, however, because of backspace and delete overrides in {productname} Core, the `beforeinput` event did not always fire when expected.
```